packages/widgetbook/test/src/widgetbook_test.dart

```diff
@@ -16,18 +17,34 @@ Matcher expectAssertionErrorWithMessage({
   );
 }
 
+GoRouter _getRouter(Widget child) {
```

Comment:
Can you please explain why this was removed from `app_builder.dart` but is used in this test?

Reply:
The Material and Cupertino app builders no longer need go_router to function correctly. I moved it to the test because the test covers a custom theme and used this implementation for the appBuilder before. We do not longer provide a default app_builder for custom theming (because in my opinion, it does not make any sense), so the test had to implement it. To keep the test the same as before, I, therefore, moved the code to the test. 

Hope that helps 😀